armi/reactor/converters/axialExpansionChanger.py

```diff
@@ -467,16 +467,14 @@ def _getLinkedComponents(self, b, c):
         lstLinkedC = [None, None]
         for ib, linkdBlk in enumerate(self.linkedBlocks[b]):
             if linkdBlk is not None:
-                for otherC in linkdBlk.getChildren():
+                for otherC in getSolidComponents(linkdBlk.getChildren()):
                     if _determineLinked(c, otherC):
                         if lstLinkedC[ib] is not None:
                             errMsg = (
-                                "Multiple component axial linkages have been found for "
-                                "Component {0}; Block {1}; Assembly {2}."
-                                " This is indicative of an error in the blueprints! Linked components found are"
-                                "{3} and {4}".format(
-                                    c, b, b.parent, lstLinkedC[ib], otherC
-                                )
+                                f"Multiple component axial linkages have been found for "
+                                f"Component {c}; Block {b}; Assembly {b.parent}."
+                                f" This is indicative of an error in the blueprints! Linked components found are"
```

Comment:
Minor thing: This line does not need the `f` before the string.

Reply:
https://github.com/terrapower/armi/pull/1331/commits/bcd4a4ab4371e471273c8115924cf7759b9ea536